examples/ref-templates/assets/htlc.py

```diff
@@ -0,0 +1,99 @@
+from pyteal import *
+
+john = Addr("2UBZKFR6RCZL7R24ZG327VKPTPJUPFM6WTG7PJG2ZJLU234F5RGXFLTAKA")
+master = Addr("WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE")
+fee = 10000
+pre_image = "QzYhq9JlYbn2QdOMrhyxVlNtNjeyvyJc/I8d8VAGfGc="
+timeout = 2000
+
+def htlc(TMPL_RCV,
+         TMPL_OWN,
+         TMPL_FEE,
+         TMPL_HASHIMG,
+         TMPL_HASHFN,
+         TMPL_TIMEOUT):
+
```

Comment:
```suggestion
```

Reply:
I think the comment has to be right below the function to qualify as a docstring